src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
So the issue with rolling generic vs proto into RpcType is that we will want generic vs proto on streaming_ping_pong, client_streaming, server_streaming, and bidi_streaming, and possibly even unary at some point. So I'd prefer to keep it a separate issue.